plugins/postcss-custom-media/README.md

```diff
@@ -123,6 +123,33 @@ instructions for:
 - [Gulp](INSTALL.md#gulp)
 - [Grunt](INSTALL.md#grunt)
 
+## CSS Modules
+
+If you're using CSS Modules, you'll probably notice that custom media queries are not being resolved. This happens
```

Comment:
Is it correct for this documentation to be tied to CSS Modules specifically? It is an issue in whatever system where CSS files are modular, i.e one might not b using CSS Modules, rather they might just be importing small CSS files in multiple places in JS. As another example, we are using vanilla-extract, which is not exactly CSS Modules, but still has the same issue and solution.

Reply:
Is there a good term that can be used to describe the issue or the way CSS is processed so that as many users as possible understand that the solution could apply to them?

Reply:
@mihkeleidast this is a good point and perhaps this is just a misunderstanding on my part. CSS Modules is not a specific package but rather a philosophy or way to handle CSS.

See https://github.com/css-modules/css-modules

If there's a better term I'm all up for changing it

Reply:
Maybe inverse it : `Modular CSS processing`

This covers `CSS Modules` but also other things like `postcss-loader` in Webpack, `vanilla-extract`, ...

Reply:
I would also use "modular css" as the general term, and then list out some of the examples of that in the next paragraph, so users can ctrl+f their specific term.